src/components/SmartLabel.brs

```diff
@@ -84,6 +84,7 @@ function addLine(text)
       poster = m.posters[part.text]
       if poster <> invalid
         newPoster = m.top.createChild("Poster")
+        newPoster.blendColor = poster.blendColor
```

Comment:
There are many other fields that the poster can have assigned (see [Poster](https://developer.roku.com/docs/references/scenegraph/renderable-nodes/poster.md#fields) Roku SDK docs). Do you see us needing to add those additional fields in the future as well?

Perhaps it would be better for us to create an array of all supported fields and assign those? Or maybe even just assign every property of `poster` to `newPoster` if we can get a list of keys from `poster`.

Thoughts @georgejecook @timalacey ?

Reply:
This change covers all my use cases so I'm happy. I doubt there's more.

> every property

would likely cause some issues, i.e. `id`